mev_inspect/aave_liquidations.py

```diff
@@ -99,3 +100,23 @@ def _get_payback_token_and_amount(
                     return child_transfer.token_address, child_transfer.amount
 
     return liquidation.inputs["_collateral"], 0
+
+
+def _get_debt_data(
+    liquidation: DecodedCallTrace, child_traces: List[ClassifiedTrace], liquidator: str
+) -> Tuple[str, int]:
+    """Get transfer from liquidator to AAVE"""
+
+    for child in child_traces:
+
+        if isinstance(child, CallTrace):
+
+            child_transfer: Optional[Transfer] = get_transfer(child)
+
+            if child_transfer is not None:
+
+                if child_transfer.from_address == liquidator:
+
+                    return child_transfer.token_address, child_transfer.amount
+
+    return liquidation.inputs["_reserve"], 0
```

Comment:
Don't we expect this transfer to always exist? Error instead?

Reply:
Wanted to confirm before throwing, since previously had no exceptions for liquidation classifiers